local/protocol/locate.go

```diff
@@ -0,0 +1,34 @@
+package protocol
+
+import (
+	"fmt"
+	"os"
+	"path"
+)
+
+// Utilities to deal with Scoot instances.
```

Comment:
I figured agreeing on how to determine the socket was part of the protocol.

Put another way: the protocol is what's the server and client have to agree on, which includes this code.